.github/workflows/test.yml

```
@@ -61,6 +61,7 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install -U pip
+          python -m pip install git+https://github.com/pytest-dev/pytest-xdist.git@7f07d50
```

```suggestion
          python -m pip install git+https://github.com/pytest-dev/pytest-xdist.git@7f07d50
```

This line should no longer be necessary or at least install a released version.